packages/react-dom/src/__tests__/ReactLegacyErrorBoundaries-test.internal.js

```diff
@@ -486,6 +487,57 @@ describe('ReactLegacyErrorBoundaries', () => {
       },
     };
 
+    BothErrorBoundaries = class extends React.Component {
```

Comment:
One of the ErrorBoundaries-test files needed to have an error boundary that used both the old and new methods, I chose the Legacy test because if Legacy is removed the test can be deleted